public/index.html

```diff
@@ -32,6 +32,7 @@
 	<link rel="preload" href="/static/font/bold.woff2" as="font" type="font/woff" crossorigin>
 	<link rel="preload" href="/static/font/medium.woff2" as="font" type="font/woff" crossorigin>
 
+	<script defer data-domain="designatuci.com" src="https://plausible.io/js/script.js"></script>
```

Comment:
Question: is specifying `data-domain` enough to avoid having development requests misclassified as actual traffic?

Reply:
Yes and no - there is a setting within plausible to turn off localhost traffic.

Reply:
It's more of a way to split up dashboards. The data-domain is mostly for this: https://plausible.io/docs/plausible-script#can-i-send-stats-to-multiple-dashboards-at-the-same-time.